charts/stable/immich/templates/_secret.tpl

```diff
@@ -22,4 +22,9 @@ data:
   {{- with .Values.immich.mapbox_key }}
   MAPBOX_KEY: {{ . | b64enc}}
   {{- end }}
+  {{- with .Values.immich.typesense_api_key }}
+  TYPESENSE_API_KEY: {{ . }}
+  {{- else }}
+  TYPESENSE_API_KEY: {{ randAlphaNum 32 | b64enc }}
```

Comment:
Why is there a random key created?

And in this case the random generated key, is not even persisted.

Reply:
The random key is created because of the note in the [documentation] that says (https://immich.app/docs/install/docker-compose#step-2---populate-the-env-file-with-custom-values)

> Consider changing TYPESENSE_API_KEY to something randomly generated